core/templates/pages/contributor-dashboard-admin-page/question-role-editor-modal/cd-admin-question-role-editor-modal.component.html

```diff
@@ -0,0 +1,128 @@
+<style>
+  .question-submitter-role-checkbox {
+    color: #00645c;
+    cursor: pointer;
+    font-size: 20px;
+  }
+  .question-role-editor-main-container {
+    background: white;
+    border-radius: 8px;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+  }
+  .question-role-editor-footer {
+    display: flex;
+    gap: 12px;
+    justify-content: end;
+    padding: 15px;
+  }
+  .question-role-editor-footer-ruler {
+    border: 1px #333 solid;
+    margin-top: 5px;
+    opacity: 0.10;
+  }
+  .cancel-button {
+    border-color: #00645c;
+    border-radius: 5px;
+    color: #00645c;
+    font-family: 'Roboto', sans-serif;
+    font-size: 14px;
+    font-weight: 500;
+    letter-spacing: 0.30px;
+    padding: 5px;
+    text-transform: uppercase;
+  }
+  .save-button {
+    background: #00645c;
+    border-radius: 5px;
+    color: white;
+    font-family: 'Roboto', sans-serif;
+    font-size: 14px;
+    font-weight: 500;
+    letter-spacing: 0.30px;
+    padding: 5px;
+    text-transform: uppercase;
+  }
+  .label {
+    color: #333;
+    font-family: 'Roboto', sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    left: 32px;
+    top: 0;
+    word-wrap: break-word
+  }
+  .label-explanation {
+    color: #667085;
+    font-family: 'Roboto', sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    word-wrap: break-word
+  }
+  .title-container {
+    color: #00645c;
+    font-family: 'Capriola', Arial, sans-serif;
+    font-size: 20px;
+    margin-bottom: 20px;
+    margin-left: 20px;
+    margin-top: 25px;
+  }
+  .sub-heading-container {
+    font-weight: 600;
+    margin-bottom: 5px;
+    margin-left: 20px;
+    word-spacing: 1.6px;
+  }
+  .label-container {
+    display: flex;
+    flex-direction: column;
+  }
+  .attribute-container {
+    display: flex;
+    padding-bottom: 5px;
+    padding-left: 20px;
+  }
+  .loading-message {
+    color: #00645c;
+    font-size: xx-large;
+    padding: 20px;
+  }
+</style>
+<div class="question-role-editor-main-container">
+  <div *ngIf="loadingMessage" class="loading-message e2e-test-loading-message">
+    <div class="oppia-align-center">
+      <span>{{ loadingMessage }}</span>
+      <loading-dots></loading-dots>
+    </div>
+  </div>
```

Comment:
Is a loading message even necessary? The modal doesn't fetch any data, and I couldn't discern the loading message in the demo.

Reply:
I was going to put it till the modal was initialized but it seems that the modal doesn't open untill initialzed. Removed it.